katdal/datasources.py

```diff
@@ -286,6 +286,8 @@ def from_url(cls, url, chunk_store='auto'):
             # Look for adjacent data directory (presumably containing NPY files)
             if chunk_store == 'auto':
                 store_path = os.path.dirname(os.path.dirname(url_parts.path))
+                if not store_path:
+                    store_path = os.path.curdir
```

Comment:
I think this still won't fix it if I'm actually in the directory with the .rdb file and just specify its base filename. I would suggest either
```python
store_path = os.path.join(os.path.dirname(url_parts.path), '..', '..')
```
or
```python
store_path = os.path.dirname(os.path.dirname(os.path.abspath(url_parts.path)))
```
(both untested)

Reply:
I like the latter.